### ISSUE TITLE
Fail on create Symlink on Windows host (dev)

### ISSUE THREAD

**Issue by AdsonCicilioti** [roadiz/roadiz#283]
I installed on Windows with Vagrant, I accessed ssh .. vm works perfectly.
I left the default IP 192.168.33.1. I copied the nginx.conf sample from the 'samples' folder to /etc/nginx/conf.d/roadiz.conf, setting the root directory to / var / www / web and server_name site1.dev and localhost.
In Windows I set up the hosts file in my domain site1.dev for ip 192.168.33.1.
I can access SOLR but dont /install.php.

I have no patience for much promise and little (or no) effectiveness. Roadiz seemed to be great, but it did not work for me.

**Comment by ambroisemaupate** [comment]
@AdsonCicilioti with Vagrant you don't need to copy anything to nginx, everything is ready to work since your VM is correctly booted. **As explained in documentation.**

GitHub issue are not meant to be a review feed. Remember that open source projects are maintained by volonteers and issues should be addressed in a constructive way to improve the project.

**Comment by AdsonCicilioti** [comment]
I had forgotten to generate the theme before creating the VM. But when trying to create I get the error 
```
[Symfony\Component\Filesystem\Exception\IOException]
   Failed to remove file "B:\dev\standard-edition/themes/AdsonTheme/.git\objects\pack-a67bf58b946e291e7319dca6fba43106fa9e17c3.pack": unlink (B:\dev\standard-edition/themes/AdsonTheme/.git\objects\pack\pack-a67bf58b946e291e7319dca6fba43106fa9e17c3.pack): Permission denied.
```

**Comment by AdsonCicilioti** [comment]
The theme was generated, all files are there.
I created the VM with vagrant, but the problem continues:

Http://192.168.33.1/install.php dont wor
Http://192.168.33.1:8983/solr WORK
Http://192.168.33.1/phpmyadmin dont work
Http://192.168.33.1:1080 WORK

**Comment by AdsonCicilioti** [comment]
Why close this issue?

**Comment by ambroisemaupate** [comment]
192.168.33.1 is your **computer** IP not your vagrant VM. 

If you are not using your VM IP you must use 8080 port. That's why Solr and Mailcatcher work but not install and phpmyadmin. 

You must define an IP for your Vagrant which **is not** 192.168.33.1 because this is your gateway.

**Comment by AdsonCicilioti** [comment]
Not at all!
Complicated to explain, but I found the solution. The problem is in the VM Linux interface running on windows, and running the composer create-project roadiz / standard-edition command, which tries to create symbolic links that do not work in the VM.
Knowing this I tried to create the symbolic links manually, from within the VM logged via SSH, but gave Protocol Error. I found the solution by calling out Virtualbox commands that enable symbolic link building in shared directories. From there I was able to create the links manually and I was able to proceed with the installation of the project. But I make it clear that the documentation is confusing and fragmented. I found some information on the Roadis site, others in the repositories on Github. I hope they improve this, because the roadiz seems to be promising. I will continue with my experiments.
Think fondly about Windows users. Hugs!
